pkg/options/patchtmpl/patch.go

```diff
@@ -75,24 +82,28 @@ func (p *parsedPatch) String() string {
 	return string(p.raw)
 }
 
-func (a *applier) makePatches(comp *bundle.Component, opts options.JSONOptions) ([]*parsedPatch, error) {
+func (a *applier) makePatches(comp *bundle.Component, opts options.JSONOptions) ([]*parsedPatch, []*unstructured.Unstructured, error) {
```

Comment:
nit: would be helpful to add a comment about this does, especially what the return types are (what are these unstructured objects?)